src/index.js

```diff
@@ -1,7 +1,46 @@
-/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
+// /* eslint-disable guard-for-in */
 import './style.css';
-import { CreatList } from './List';
+// import { Task } from './Task';
+// eslint-disable-next-line import/no-cycle
+import { TaskList } from './tasklist';
 
-const Tasks = [{ description: 'Task1', completed: true, index: 1 }, { description: 'Task2', completed: false, index: 2 }, { description: 'Task3', completed: false, index: 3 }];
-// eslint-disable-next-line no-restricted-syntax
-CreatList(Tasks);
+// const NewTaskList = new TaskList(JSON.parse(localStorage.getItem('library')) || []);
+// NewTaskList.ShowTasks();
+
+// localStorage.clear();
+// function addTasks() {
+//   const TaskText = document.getElementById('TaskInput');
+//   const TaskId = Math.floor(Math.random() * 1000); // idGenerate();
+//   const NewTask = new Task(TaskText.value, false, TaskId);
+//   TaskText.value = '';
+//   NewTaskList.addTask(NewTask);
+//   NewTaskList.AddToStorage();
+// }
+// eslint-disable-next-line max-len
+const NewTaskList = new TaskList([]);
```

Comment:
- [ ] Kindly remove all the commented code from your project as they occupy unnecessary space and clutter your work.
- [ ] [OPTIONAL] Also, try to solve the linter errors you get instead of disabling them as they are there to help you write clean code.